integrations/tensorflow/bindings/python/pyiree/tf/support/tf_test_utils.py

```diff
@@ -427,6 +446,22 @@ def serialize(self, trace_dir: str) -> None:
       call_dir = os.path.join(trace_dir, f"call_{str(i).zfill(width)}")
       call.serialize(call_dir)
 
+    # C++ Serialization.
+    if "tf" not in self.backend:
+      flagfile = []
+      if self.compiled_path is not None:
+        # Can be overridden with another flag after the flagfile.
```

Comment:
That's true of all flags for a flagfile

Reply:
That's fair. I thought that overriding the compiled path warranted noting since it's something we want to be able to do, but that comment belongs in a readme, not in the code.